apps/frontend/src/store/fileBrowserStore.ts

```diff
@@ -478,7 +478,26 @@ export const deselectAll = () => {
     selectedPaths.value = [];
 };
 
-export const itemAction = async (item: PathInfoModel, path = currentDirectory.value) => {
+export const handleAllSelect = (val: any) => {
```

Comment:
Is it possible to just pass the value of the checked to this function? so no any is needed


where you are using handleAllSelected, instead of passing the target, pass the target.checked prop